core/components/atoms/input/__stories__/variants/types/WithLabel.story.jsx

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import { Input, Text, Label, HelpText } from '@/index';
 import { action } from '@/utils/action';
+import './style.css';
 
 // CSF format story
 export const withLabel = () => {
   return (
     <div className="Row">
       <div className="mr-9 mb-8 w-25">
-        <div style={{ height: '72px' }}>
+        <div className="Icon-labelLevel">
```

Comment:
Please remove this custom className. We can move "No Help Text" message above the Input or find any other way instead of using fixed height